examples/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "examples",
+    "version": "0.4.0-test.0",
+    "private": true,
```

Comment:
'examples' is a private package in the monorepo, it consumes @locker/dom-membrane